gradle.properties

```diff
@@ -26,11 +33,11 @@ developmentEnvironmentUserName = Developer
 # The string's content will be replaced with your mod's version when compiled. You should use this to specify your mod's
 # version in @Mod([...], version = VERSION, [...])
 # Leave these properties empty to skip individual token replacements
-replaceGradleTokenInFile = BinnieCore.java
```

Comment:
the gradletoken updates need to be reverted; they are incorrect.  @Dream-Master please let someone familiar with these things review and approve before merging 😡